.github/workflows/CI.yml

```diff
@@ -247,6 +247,20 @@ jobs:
         project_exe_stem=${PROJECT_EXES%% *}
         cp "src/${project_exe_stem}${{ steps.vars.outputs.EXE_suffix }}" "${{ steps.vars.outputs.PKG_DIR }}/bin/${project_exe_stem}-gui${{ steps.vars.outputs.EXE_suffix }}"
 
+    - name: "Build WinOS text+gui hybrid"
+      if: runner.os == 'Windows' && steps.vars.outputs.STATIC != 'true' ## WinOS, non-static (unable to build static gtk/gui)
+      shell: bash
+      run: |
+        # create and stage text+gui hybrid for Windows
+        # * copy only main/first project binary
+        project_exe_stem=${PROJECT_EXES%% *}
+        # * clean/remove build artifact(s)
+        rm "src/${project_exe_stem}${{ steps.vars.outputs.EXE_suffix }}" ##.or.# opam exec -- make -C src clean #.or.# opam exec -- make clean
+        # * re-create (with hybrid text+gui UI)
+        opam exec -- make src OSTYPE=$OSTYPE UISTYLE=gtk3 UI_WINOS=hybrid STATIC=${{ steps.vars.outputs.STATIC }}
+        project_exe_stem=${PROJECT_EXES%% *}
```

Comment:
nit: `project_exe_stem=${PROJECT_EXES%% *}` is duplicated a couple of rows above.

Reply:
Removed it; thanks!